styles.css

```diff
@@ -0,0 +1,466 @@
+@import url("https://fonts.googleapis.com/css2?family=Play&display=swap");
```

Comment:
Consider adding more comments in your CSS file for clarity on what you are trying to achieve, and group your CSS within selectors for clearer organisation of which CSS properties are trying to achieve what.